lib/internal/bootstrap/pre_execution.js

```diff
@@ -250,8 +250,11 @@ function setupFetch() {
       }
 
       // Pass all data from the response body to the WebAssembly compiler.
-      for await (const chunk of response.body) {
-        streamState.push(chunk);
+      const { body } = response;
```

Comment:
Separately, it would be nice to just have this entire function in another JS file and load it into pre_execution.js to keep the routine focusing on bootstrap.

Reply:
SGTM, will do that once this lands.

Reply:
See https://github.com/nodejs/node/pull/42993.